### DISCUSSION TITLE
How to contribute and collaborate on Thinkster?

### DISCUSSION THREAD

**Discussion by yaten2302** [realworld-apps/realworld#1548] [Q&A] [answered]
Hi @geromegrignon @mvila, I'm Yaten, I want to contribute to the existing repos that are currently being developed. Could you please guide that are we allowed to collaborate with others and work on a single project? Also, like what kind of projects are accepted? Like in the discussions, I saw that `Announce your work in progress implementation here`, could you please explain something about this🙏🏼
Also, is there some discord channel or slack for Thinkster, where we can discuss and collaborate with others?

**Comment by geromegrignon** [comment]
Hi @yaten2302, I'll provide you a complete answer in a few days as being busy right now.
While @TaiTaiTai777 message might be useful, it's unrelated directly to this project (and more likely AI generated).

**Reply by yaten2302** [reply]
Sure, no issues. Feel free to answer whenever you are free👍

**Comment by geromegrignon** [comment]
Hi @yaten2302,

Sorry for the extended delay. Currently we do not provide help about collaboration on wip implementations. There were attempts but most projects are created and implemented by one person. The `Announce your work in progress implementation here` is our most simplest way to know if projects are ongoing and the easiest solution for you to contact someone.

However, mind a lot of projects listed there are never finished. If you are interested in helping, contact the maintainer directly to propose your help.

We do not have Discord/Slack, the Discussions feature being the perfect fit as we do not have that much traffic.

About projects accepted, they must follow the specs from your documentation: https://www.realworld.how/docs/implementation-creation/expectations
We don't accept duplicate projects or small derivations from an existing accepted implementation. Feel free to ask questions if you have any doubt.

**Reply by yaten2302** [reply]
Hey! @geromegrignon, thanks for your response 👍. Actually, I've a doubt, that what kind of projects are accepted here? Like, I saw a repo of a full stack medium clone with - React, Angular... So, like we've to create only medium clones with different tech stack or it's different?

**Reply by geromegrignon** [reply, marked as answer]
Yes, that's exactly about Medium clones.
